PSOperations.podspec

```diff
@@ -36,7 +36,58 @@ Pod::Spec.new do |s|
 	s.osx.deployment_target = "10.11"
 
 	s.requires_arc = true
+	s.pod_target_xcconfig = { "EMBEDDED_CONTENT_CONTAINS_SWIFT" => "YES" }
 
 	s.source 	= {  git: "https://github.com/pluralsight/PSOperations.git",  tag: s.version.to_s  }
-	s.source_files = 'PSOperations/**/*.swift'
+	
+	subspec_sources = { 
+		"CloudKit" => [
+			"PSOperations/iCloudContainerCapability.swift", 
+			"PSOperations/CloudCondition.swift",
+			"PSOperations/CKContainer+Operations.swift",
+			"CloudCondition.swift"
+		],
+		"HealthKit" => ["PSOperations/Health*.swift"],
+		"CoreLocation" => ["PSOperations/Location*.swift"],
+		"PassKit" => ["PSOperations/Passbook*.swift"],
+		"Photos" => ["PSOperations/Photos*.swift"],
+		"SystemConfiguration" => ["PSOperations/ReachabilityCondition.swift"],
+		"EventKit" => ["PSOperations/Calendar*.swift"]
+		}
+	
+	s.source_files = "PSOperations/**/*.swift"
+	s.exclude_files = subspec_sources.values.flatten()
+
+	s.subspec "Core" do |sb|
+		sb.source_files = "PSOperations/**/*.swift"
+		sb.exclude_files = subspec_sources.values.flatten()
+	end
+
+	s.subspec "CloudKit" do |sb|
```

Comment:
Since Ruby is a full turing language, you could iterate over the subspec_sources hash and create subspecs from that data.